src/api/mocks/testdata/repo-incubator.yaml

```diff
@@ -1,25 +1,40 @@
 apiVersion: v1
```

Comment:
The stable repo index has been updated with the correct order so you may want to update it, but not absolutely necessary for Monocular.